### DISCUSSION TITLE
Add group query param

### DISCUSSION THREAD

**Discussion by metalmarco** [directus/directus#3205] [Feature Requests]
In v8 we were able to specify in the get items request the "groups" parameter, which allowed to group results and get unique values.

In v9 the groups parameter seems to be unavailable.

---

### Project details
Directus Version: 9.0.0-rc.18
Environment: production
OS: Linux 5.4.0-1029-aws
Node: 14.15.1

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
